### PR TITLE
blog info looks prettier

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

**Comment by vied12** [comment]
LGTM

File: client/app/scripts/liveblog-bloglist/views/main.html [hunk 1]
```diff
@@ -64,12 +64,16 @@
...
-                        <dt>{{ ::'Number of Posts' | translate }}:</dt>
+                        <dt>{{ ::'No. of Posts' | translate }}:</dt>
                         <dd>{{ ::blog.posts_count }}</dd>
                         <dl ng-if="blog.last_posted">
                         <dt translate> Last posted</dt>
                         <dd sd-reldate ng-model="blog.last_posted"></dd>
                         </dl>
+                        <dl ng-if="blog.posts_count == 0">
+                        <dt translate> Last posted</dt>
+                        <dd> No posts yet</dd>
```
Comment (vied12) [inline client/app/scripts/liveblog-bloglist/views/main.html]: Why don't you put `ngIf` on this `dd` line instead of all the `dl` ?

Comment (ancafarcas) [inline client/app/scripts/liveblog-bloglist/views/main.html]: This syntax was "friendly" to me and I didn't want to waste time on finding another one. Is it looking awful?

Comment (vied12) [inline client/app/scripts/liveblog-bloglist/views/main.html]: Absolutly not awful !
It's just shorter and avoid repetition:
```html
<dl>
    <dt translate> Last posted</dt>
    <dd ng-if="blog.last_posted" sd-reldate ng-model="blog.last_posted"></dd>
    <dd ng-if="blog.posts_count == 0"> No posts yet</dd>
</dl>
```

Comment (ancafarcas) [inline client/app/scripts/liveblog-bloglist/views/main.html]: Thanks :)

**Pull request merged**